src/cardSystem/Field.py

```diff
@@ -0,0 +1,35 @@
+import pygame
+from src.dependency import *
+
+class Field:
+    def __init__(self, index, position):
+        self.index = index  # Index in the row
+        self.position = position  # (x, y) coordinates for rendering
+        self.entity = None  # Store the entity occupying the field
+
+    def is_occupied(self):
```

Comment:
@rymTooo ตรงนี้กูงงว่ะเพื่อน อะไรงี้

วิธีการทำก็กดที่ File change -> จิ้มบรรทัดที่สนใจจะมีปุ่ม + -> พิมคอมเม้นไว้

Reply:
I don't pim thai in pc na. I can't a hahahah

Reply:
We implemented it on entity class and the plac_entity function below